libs/data-access/api/src/lib/trpc/routers/learning-diary.router.ts

```diff
@@ -17,7 +17,7 @@ export const learningLocationRouter = t.router({
 		return database.learningLocation.create({
 			data: {
 				name: input.name,
-				iconURL: input.iconURL,
+				iconURL: input.iconURL ?? "",
```

Comment:
warum nicht null wie vorher?

Reply:
Die DB oder TRCP habe da Probleme mit,

Reply:
kannst du das mehr beschreiben? Ich verstehe das noch nicht

Reply:
ich hätte jett erwartet, dass man dann auch in die Datenbank null rein schreiben kann

Reply:
Ich schaue mir mal an, ob ich das auch noch anders gelöst bekomme.

Reply:
Hab das Ganze im z Schema auf nullable gesetzt. Jetzt darf man da wieder null herein tun.